index.template.html

```diff
@@ -1223,29 +1223,31 @@ <h2>Core Vocabulary Definitions</h2>
       %s
 
       </section>
-    
+
       <section id="sec-data-schema-vocabulary-definition">
       <h2>Data Schema Vocabulary Definitions</h2>
 	  <p>A data schema is an abstract notation for data contained in data formats.</p>
       <p>
-        The data schema vocabulary definition reflects a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]]. It is noted
+        The data schema vocabulary definition reflects a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]].
+        A JSON Schema [[?JSON-SCHEMA]] processor for <a href="https://json-schema.org/specification-links.html#draft-7">JSON Schema draft 7</a>
+        can consume a Data Schema. It is noted
```

Comment:
```suggestion
        can consume a data schema. It is noted
```

It seems to be lowercase everywhere else...

Reply:
Thank you :)